src/Compilers/Core/MSBuildTask/Microsoft.VisualBasic.Core.targets

```diff
@@ -1,6 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="DiscoverEditorConfigFiles" Condition="'$(DiscoverEditorConfigFiles)' != 'false'" BeforeTargets="CoreCompile">
```

Comment:
Not at the moment.

@jaredpar I think we should have a discussion about how coupled we want these to be going forward. Having a single build task was a blocker to having separate Csc/Vbc compiler NuGet packages. I think we should decide whether we want to split these up once and for all or just accept that we'll always ship them together.